src/js/containers/covid19/awardSpendingAgency/AwardSpendingAgencyTableContainer.jsx

```diff
@@ -293,6 +293,12 @@ const AwardSpendingAgencyTableContainer = (props) => {
         window.scrollTo(0, 0);
     }, [document]);
 
+    useEffect(() => {
+        if (request.current) {
+            request.current.cancel();
+        }
+    }, [request]);
```

Comment:
This is gonna fire too often. What we want is for this `useEffect` to fire right before the component unmounts. Take a look at the react documentation on use effect. You should be able to find this there!